### PR TITLE
Tagging the first release + N/A for CPT/Dapsone should not require other data to be entered 

### PR DESCRIPTION
**Pull request by ssmusoke**
1. https://trello.com/c/ofZSRAXm

2. Add fix for directions to residence concept on the summary page which was a drop down instead of a text field https://trello.com/c/8A6VNOuU

### CODE REVIEW

**Comment by slubwama** [comment]
@ssmusoke kindly find a way of having one commit for your PRs. This is getting out of hand

File: pom.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 	<groupId>org.openmrs.module</groupId>
 	<artifactId>ugandaemr-iss</artifactId>
-	<version>1.0.0-SNAPSHOT</version>
+	<version>1.0.0</version>
```
Comment (slubwama) [inline pom.xml:13]: @ssmusoke isn't this supposed to change with a release instead of just editing

Comment (ssmusoke) [reply pom.xml:13]: @slubwama This is not pushed to the OpenMRS repository - hence editing it manually. It is the same thing

**Comment by ssmusoke** [comment]
@slubwama 

1. See new fix on the directions to residence which cannot be put in a separate PR due to potential merge conflicts

2. Squashing commits does not add any value since you can do that when merging the PR into master. That option was not available before hence the OpenMRS push to squash not to pollute master with many commits

**Comment by slubwama** [comment]
@ssmusoke  I think a PR is smarter when it has one commit. I am not saying that there is not option for that but I think It's better for a PR to have one commit. This has 42. 
Here is how you can keep your PR with one commit. 
1. Always Create a branch from upstream other than local master. 
2. Keep your forked master updated. 
3. Rebase your commits to keep them one commit for a PR. 
That way Tracking becomes very easy in case there is  need roll back. Rolling Back this Pr may become  problematic if any issues a-raise from merge.

**Comment by ssmusoke** [comment]
@slubwama I totally agree with you, however at times it is not possible so it should be okay...

**Comment by slubwama** [comment]
@ssmusoke  This seems not solved. 
screenshot 2019-03-05 at 11 39 51

**Comment by slubwama** [comment]
> @ssmusoke This seems not solved.
> screenshot 2019-03-05 at 11 39 51

@ssmusoke  this is resolved. Form was not just Being deployed

**Pull request merged**